.azure/pipelines/ci.yml

```diff
@@ -112,6 +112,16 @@ jobs:
             $(_BuildArgs)
       displayName: Build Installers
 
+    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
The place for this is in https://dev.azure.com/dnceng/internal/_release?view=mine&definitionId=31. That's temporary but consistent with our other push operations for now.

Reply:
@dougbu 
I'm good with any placement that your team would prefer - as long as we do this soon. Can we get this up and running middle of next week? What do you think?

Reply:
@dougbu what do we need to do to add this to the aspnetcore-cd-official feed?

Reply:
somebody needs to update that release pipeline. if nobody else does it first, I'll get to it today or tomorrow

Reply:
I have no idea how to update the release pipeline. If you could do that, it would be much appreciated.